bot/pagination.py

```diff
@@ -62,23 +73,33 @@ def add_line(self, line: str = '', *, empty: bool = False) -> None:
         """
         Adds a line to the current page.
 
-        If the line exceeds the `self.max_size` then an exception is raised.
+        If the line exceeds `self.max_size`, then `self.max_size` will go up to `scale_to_size` for
+        a single line before creating a new page. If it is still exceeded, the excess characters
+        are stored and placed on the next pages until there are none remaining (by word boundary).
+
+        Raises a RuntimeError if `self.max_size` is still exceeded after attempting to continue
+        onto the next page.
```

Comment:
Oops, this still needs to be updated to mention that it truncates the line instead of raising a `RuntimeError`. Heading to sleep now, but I should be able to find the time to make a quick fix tomorrow.